proxy.go

```diff
@@ -73,7 +73,7 @@ func Join(local, remote net.Conn, log logging.Logger) {
 
 	transfer := func(side string, dst, src net.Conn) {
 		log.Debug("proxing %s -> %s", src.RemoteAddr(), dst.RemoteAddr())
-
+		defer dst.Close()
```

Comment:
Hi @rjeczalik, thanks for the reply! I'm fairly new to Golang, so I apologize if this wasn't the most optimal solution. 

I took a look at the docs for CloseRead() and they say "Most callers should just use Close." Should I still change this line to CloseRead anyway? 

@cihangir The second iteration of the transfer function hangs on line 77 for presumably the same reason the browser hangs. 

I will be adding some additional context below.